partner_attribute_hls/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2021 Quartile Limited
+{
+    "name": "Partner Attribute HLS",
+    "version": "12.0.1.0.0",
+    "category": "Contact",
+    "website": "https://www.quartile.co/",
+    "author": "Quartile Limited",
+    "license": "Other proprietary",
```

Comment:
```suggestion
    "license": "LGPL-3",
```
Add the heading comment as well.